docs/quickstarts/word-quickstart.md

```diff
@@ -284,7 +284,7 @@ Congratulations, you've successfully created a Word task pane add-in! Next, lear
 
 ### Try it out
 
-1. Using Visual Studio, test the newly created Word add-in by pressing **F5** or choosing the **Start** button to launch Word with the **Show Taskpane** add-in button displayed in the ribbon. The add-in will be hosted locally on IIS.
```

Comment:
Why remove the part about pressing the Start button?

Reply:
Because this instruction is incomplete, if you create your first add-in the add-in is not ' just' sideloaded by starting Word using the Start menu, launching Word. For that you need to do a (for a beginner hard to accomplish) sideloading procedure loading the Manifest first. If you just hit F5 in VS this all will be managed by VS..
The removed text *does* work if you previously ran the add-in using VS F5 but the text implies this not to be the case.

Reply:
Just for kicks I also tested the First Add-in in my "Blazor" project and is now working over there as well😎
https://blazortestaddin.azurewebsites.net
You need to side-load the manifest:
https://github.com/aafvstam/BlazorAddIn/blob/master/BlazorAddIn/manifest.xml

Have fun!

Reply:
Maybe we're talking about different starting options? Fully willing to admit I'm mistaken, but isn't F5 just the shortcut key for this Start button? I just created a brand new Word add-in through VS and clicking that button launched Word with the ribbon button visible.

![image](https://user-images.githubusercontent.com/38896772/155614043-b72dd3a1-b101-47ce-98a5-f4753cb7d904.png)

Reply:
Oooh ok 😂 wasn't thinking about that Start button. In that case maybe the wording is confusing, at least I didn't get the pointer there 😁. You can leave the text in if you like obviously, only remove the semicolon issues.

Reply:
I've added a suggestion for users to do "Debug" then "Start Debugging"